services-custom/dynamodb-enhanced/README.md

```diff
@@ -152,7 +152,7 @@ index. Here's an example of how to do this:
    ```java
    DynamoDbIndex<Customer> customersByName = customerTable.index("customers_by_name");
        
-   PageIterable<Customer> customersWithName = 
+   SdkIterable<Page<Customer>> customersWithName = 
```

Comment:
Can we add the following code which converts a `SdkIterable` to `PageIterable` after this line?

```
PageIterable<Customer> pages = PageIterable.create(customersWithName);
```